source/dub/internal/vibecompat/inet/url.d

```diff
@@ -107,7 +107,7 @@ struct URL {
 			}
 		}
 
-		this.localURI = str;
+		this.localURI = (str == "") ? "/" : str;
```

Comment:
I would think this would be more natural to be handled when appending, but I noticed that it's not possible to append an absolute path, for some reason. So perhaps it doesn't matter.